test/test-integration/src/test/resources/apis/openApis/mockConsulApis.yaml

```diff
@@ -0,0 +1,720 @@
+basePath: /v2
```

Comment:
Shall we remove the unused resources?
For example resources like `/user` , `/user/login` etc you can remove